utils/gui/Minimap.js

```diff
@@ -0,0 +1,164 @@
+import PlanarView from 'Core/Prefab/PlanarView';
+import Coordinates from 'Core/Geographic/Coordinates';
+import { MAIN_LOOP_EVENTS } from 'Core/MainLoop';
+
+
+const DEFAULT_OPTIONS = {
+    minAltitude: 5E6,
+    maxAltitude: 30E6,
+    width: 100,
+    height: 100,
+    position: 'bottom-left',
+};
+
+
+/**
+ * A widget for minimap
+ *
+ * @property    {HTMLElement}   domElement      An html div containing the minimap.
+ * @property    {HTMLElement}   parentElement   The parent HTML container of `this.domElement`.
+ */
+class Minimap {
+    /**
+     * @param   {View}                  view                                    The iTowns view the minimap should be
+                                                                                * linked to.
+     * @param   {ColorLayer}            layer                                   The {@link ColorLayer} that should be
+                                                                                * displayed on the minimap.
+     * @param   {Object}                [options]                               The minimap optional configuration.
+     * @param   {HTMLElement}           [options.parentElement=view.domElement] The parent HTML container of the div
+                                                                                * which contains minimap widgets.
+     * @param   {number}                [options.minAltitude=5E6]               The minimal altitude the minimap camera
+                                                                                * can reach.
+     * @param   {number}                [options.maxAltitude=5E6]               The minimal altitude the minimap camera
```

Comment:
could replace by max zoom or min zoom?

Reply:
I introduced a maximal and minimal pixel size instead.